instrumentation/jedis/jedis-3.0/src/main/java/io/opentelemetry/auto/instrumentation/jedis/v3_0/JedisInstrumentation.java

```diff
@@ -44,7 +45,7 @@
 public final class JedisInstrumentation extends Instrumenter.Default {
 
   public JedisInstrumentation() {
-    super("jedis", "redis");
+    super("jedis", Constants.REDIS);
```

Comment:
this happens to match the constant, but is not really related to db_system
```suggestion
    super("jedis", "redis");
```